modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java

```diff
@@ -0,0 +1,310 @@
+    private static final String EMAIL_VALUE = "email@wso2.com";
+    private static final String OPENID_SCOPE_PROPERTY = "openid";
+    private static final String OPENID_SCOPE_RESOURCE = "/_system/config/oidc";
+
+    private String consumerKey;
+    private String consumerSecret;
+
+    private DefaultHttpClient client;
+
+    private static final String FIRST_NAME_CLAIM_URI = "http://wso2.org/claims/givenname";
+    private static final String LAST_NAME_CLAIM_URI = "http://wso2.org/claims/lastname";
+
+    private static final String USERNAME = "oauthuser";
+    private static final String PASSWORD = "oauthuser";
+
+    private PropertiesAdminServiceClient propertiesAdminServiceClient;
+    private ClaimMetadataManagementServiceClient claimMetadataManagementServiceClient;
+
+    private String openidScope = "sub,email,email_verified,name,family_name,given_name,middle_name,nickname," +
+            "preferred_username,profile,picture,website,gender,birthdate,zoneinfo,locale,updated_at,phone_number,phone_number_verified," +
+            "address,street_address,country,formatted,postal_code,locality,region";
+
+    @BeforeClass(alwaysRun = true)
+    public void testInit() throws Exception {
+        super.init(TestUserMode.TENANT_USER);
+
+        setSystemproperties();
+        client = new DefaultHttpClient();
+
+        remoteUSMServiceClient.addRole(OAUTH_ROLE, null, null);
+        remoteUSMServiceClient.addUser(USERNAME, PASSWORD,
+                null, getUserClaims(),
+                "default", false);
+        propertiesAdminServiceClient = new PropertiesAdminServiceClient(backendURL, sessionCookie);
+
+        propertiesAdminServiceClient.updateProperty(OPENID_SCOPE_RESOURCE, OPENID_SCOPE_PROPERTY, openidScope + "," + OIDC_ROLE_CLAIM_URI
+                , OPENID_SCOPE_PROPERTY);
+
+        claimMetadataManagementServiceClient = new ClaimMetadataManagementServiceClient(backendURL, sessionCookie);
+
+        ExternalClaimDTO externalClaimDTO = new ExternalClaimDTO();
+        externalClaimDTO.setExternalClaimDialectURI(OIDC_DIALECT_URI);
+        externalClaimDTO.setExternalClaimURI(OIDC_ROLE_CLAIM_URI);
+        externalClaimDTO.setMappedLocalClaimURI(ROLE_CLAIM_URI);
+        claimMetadataManagementServiceClient.addExternalClaim(externalClaimDTO);
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void atEnd() throws Exception {
+        deleteApplication();
+        propertiesAdminServiceClient.updateProperty(OPENID_SCOPE_RESOURCE, OPENID_SCOPE_PROPERTY, openidScope, OPENID_SCOPE_PROPERTY);
+
+        remoteUSMServiceClient.deleteRole(OAUTH_ROLE);
+        remoteUSMServiceClient.deleteUser(USERNAME);
+        claimMetadataManagementServiceClient.removeExternalClaim(OIDC_DIALECT_URI, OIDC_ROLE_CLAIM_URI);
+
+        consumerKey = null;
+    }
+
+    @Test(groups = "wso2.is", description = "Check Oauth2 application registration")
+    public void testRegisterApplication() throws Exception {
```

Comment:
Other methods have a new line at starting of the method. So it would be consistent if you follow the same way for all methods.